src/components/IconTable/index.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import { css } from 'emotion';
+
+interface Icon {
+  src: string;
+  left: number;
+  top: number;
+  width: number;
+  height: number;
+  onMouseEnterHandler?: (event?: React.MouseEvent) => void;
+  onMouseExitHandler?: (event?: React.MouseEvent) => void;
+  onClickHandler?: (event?: React.MouseEvent) => void;
+}
+
+interface IconTableProps {
+  heightVW: number;
+  backgroundSrc: string;
+  icons: Icon[];
+}
+
+export default class IconTable extends React.Component<IconTableProps> {
+  constructor(props) {
+    super(props);
+  }
+
+  drawIcons() {
+    const {icons} = this.props;
+
+    return icons.map(({
+      src, left, top, width, height,
+      onClickHandler = () => {},
+    }) => {
+
+      return (<div className={css`
+          position: absolute;
+          width: ${width}vw;
+          height: ${height}vw;
+          margin-left: ${left}vw;
+          margin-top: ${top}vw;
+        `}
+          onMouseEnter={(e) => e.preventDefault()}
+        >
+        <svg width="100%" height="100%"
```

Comment:
Wait if src is an svg why are we putting it in another svg?

Reply:
I think it was a leftover for my experiment yesterday. Could simplify it later on